tests/base/test_pickling.py

```diff
@@ -0,0 +1,26 @@
+import pickle
+import unittest
+
+import numpy as np
+
+from sktime.markovprocess import MaximumLikelihoodMSM
+
+
+class TestBase(unittest.TestCase):
+
+    def test_pickle_msm(self):
+        msm = MaximumLikelihoodMSM(lagtime=1, dt_traj='4 ps')
```

Comment:
nice! perhaps we can automatize this test a little by creating a method that takes an instance of a model (the test's parameter) so wed just have to have a model factory (`estimator type->model`) which is i think also useful for other tests

Reply:
Most of the time models are created in a very specific way in order to test certain things. Can you please elaborate a bit where you think this factory might be of good use?